docs/data_services.yaml

```diff
@@ -0,0 +1,143 @@
+openapi: 3.1.0
+info:
+  title: data_services
+  version: '1.0'
+  summary: Create data services
+servers:
+  - url: 'http://data-catalogue-staging.heroku.com'
+paths:
+  /apis/v1/data_services:
+    post:
+      summary: Create New User
+      operationId: post-data-service
+      responses:
+        '201':
+          description: Data Service Created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/data_service'
+              examples:
+                New Data Service Citizen API:
+                  value:
+                    identifier: abcd1234
+                    type: DataService
+                    title: Citizen API
+                    description: Provides access to DWP citizen information to Other Government Departments and approved external agencies.
+                    keywords: dwp, citizen
+                    theme: population
+                    license: ISC
+                    version: 1
+                    accessRights: OPEN
+                    issued: 2-2020
+                    contactPoint:
+                      contactName: John Doe
+                      email: john.doe@example.com
+                    modified: 10-1-2023
+                    creator: department-for-work-pensions
+                    published: 10-5-2022
+        '400':
+          description: Missing Required Information
+      description: Create a new data service.
+    parameters: []
+components:
+  schemas:
+    data_service:
+      title: data_service
+      type: object
+      properties:
+        identifier:
+          type: string
+        type:
+          type: string
+          enum:
+            - DataService
+            - DataSet
+        title:
+          type: string
+        description:
+          type: string
+        keywords:
+          type: array
```

Comment:
In the JSON schema the `keyword`, `theme` and `contactPoint` are defined as `oneOf` rather than being a single type. It seems we can do that with OpenAPI too at least for the keyword and theme: https://github.com/Redocly/redoc/issues/1764#issuecomment-970241534